packages/eslint-plugin/src/rules/prefer-regexp-exec.ts

```diff
@@ -0,0 +1,120 @@
+/**
```

Comment:
Actually sorry we've just changed to using all-contributors now and removed all other occurrences of these file-level author comments, please could you remove this